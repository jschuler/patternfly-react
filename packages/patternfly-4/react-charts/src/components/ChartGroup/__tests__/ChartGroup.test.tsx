// Auto-generated do not edit

import * as React from 'react';
import { shallow } from 'enzyme';
import { ChartGroup } from '../ChartGroup';
import { ChartContainer } from '../../ChartContainer';

describe('ChartGroup test', () => {
  it('ChartGroup should match snapshot', () => {
    const view = shallow(
      <ChartGroup
        allowZoom={false}
				animate={undefined}
				ariaDesc={undefined}
				ariaTitle={undefined}
				categories={undefined}
				children={undefined}
				color={undefined}
				colorScale={undefined}
				containerComponent={<ChartContainer />}
				data={undefined}
				domain={undefined}
				domainPadding={undefined}
				eventKey={undefined}
				events={undefined}
				externalEventMutations={undefined}
				groupComponent={undefined}
				height={undefined}
				horizontal={undefined}
				labelComponent={undefined}
				labels={undefined}
				maxDomain={undefined}
				minDomain={undefined}
				name={undefined}
				offset={undefined}
				origin={undefined}
				padding={undefined}
				polar={undefined}
				range={undefined}
				samples={undefined}
				scale={undefined}
				sharedEvents={undefined}
				singleQuadrantDomainPadding={undefined}
				sortKey={undefined}
				sortOrder={undefined}
				standalone={undefined}
				style={undefined}
				theme={{}/*unrecognizedType ChartThemeDefinition undefined*/}
				themeColor={undefined}
				themeVariant={undefined}
				width={undefined}
				x={undefined}
				y={undefined}
				y0={undefined}
      />);
    expect(view).toMatchSnapshot();
  });
});
