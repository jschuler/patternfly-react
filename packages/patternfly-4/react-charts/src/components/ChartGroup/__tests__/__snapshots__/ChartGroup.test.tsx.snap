// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ChartGroup test ChartGroup should match snapshot 1`] = `
<VictoryGroup
  containerComponent={
    <Unknown
      className="pf-c-chart"
      theme={Object {}}
    />
  }
  groupComponent={<g />}
  samples={50}
  sortOrder="ascending"
  standalone={true}
  theme={Object {}}
/>
`;
